spec/01-unit/01-db/01-schema/01-schema_spec.lua

```diff
@@ -3787,6 +3787,75 @@ describe("schema", function()
     end)
   end)
 
+  describe("get_constraints", function()
+    it("returns empty constraints", function()
+      local test_schema = {
+        name = "test",
+        fields = { { name = { type = "string" }, }, },
+      }
+
+      local TestEntities = Schema.new(test_schema)
+      local constraints = TestEntities:get_constraints()
+
+      assert.are.same({}, constraints)
+    end)
+
+    it("returns constraints", function()
+      local schema1 = {
+        name = "test1",
+        fields = { { name = { type = "string" }, }, }
+      }
+      local schema2 = {
+        name = "test2",
+        fields = {
+          { foreign_reference1 = { type = "foreign", reference = "test1" } },
+        },
+      }
+      local schema3 = {
+        name = "test3",
+        fields = {
+          { foreign_reference2 = { type = "foreign", reference = "test1", on_delete = "cascade" } },
+        },
+      }
+
+      local Entities1 = Schema.new(schema1)
+      assert.is.Truthy(Entities1)
+      local Entities2 = Schema.new(schema2)
+      assert.is.Truthy(Entities2)
+      local Entities3 = Schema.new(schema3)
+      assert.is.Truthy(Entities3)
+      local constraints = Entities1:get_constraints()
+
+      assert.are.same({
```

Comment:
This test is failing because the `constraints` list has the items in a different order - is there some way to assert unordered arrays are the same? What is the best pattern to do that?

Reply:
i typically sort the array before comparing.